src/app/components/client/PetitionBanner.module.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../tokens";
+
+.banner {
+  background: linear-gradient(90deg, $color-purple-90 50%, $color-purple-60);
+  display: flex;
+  justify-content: center;
+  padding: $layout-xs $layout-sm;
+  position: relative;
+}
+
+.content {
+  h2 {
+    font: $text-title-xs;
+    font-weight: 500;
+  }
+
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-xs;
+  color: $color-white;
+  max-width: $content-lg;
+}
+
+.buttons {
+  display: flex;
+  gap: $spacing-xs;
+  margin-top: $spacing-md;
```

Comment:
Nit since it's already here, but as a habit I'd prefer a `gap` on the parent (and wrapping the other elements that need a smaller gap in their own element), just to make future layout changes more predictable.

Reply:
I’ve went back and forth on this one: Setting `gap` on the parent would also mean we would need to [wrap these children of the content](https://github.com/mozilla/blurts-server/pull/4711/files#diff-52d2859254183b0024f3e12fb42b18c39e15e8145276de55ad07423c27d97824R29-R36) in an additional `div` which I decided against.